cocos/renderer/CCTextureCache.cpp

```diff
@@ -31,6 +31,7 @@ THE SOFTWARE.
 #include <stack>
 #include <cctype>
 #include <list>
+#include <future>
```

Comment:
Why should include `future` header file?
Is there anywhere we using `future` functionalities in this Pull Request?